web-client/src/presenter/computeds/formattedDocketEntries.ts

```diff
@@ -343,12 +338,56 @@ export const formattedDocketEntries = (
         userAssociatedWithCase,
         visibilityPolicyDateFormatted,
       });
+    })
+    .map(docketEntry => {
+      const computeMinuteEntry = (rawDocketEntry: RawDocketEntry): boolean => {
+        const MINUTE_ENTRIES_EVENT_CODES = Object.keys(
+          MINUTE_ENTRIES_WITHOUT_DOCS_MAP,
+        ).map(key => MINUTE_ENTRIES_WITHOUT_DOCS_MAP[key].eventCode);
+
+        return MINUTE_ENTRIES_EVENT_CODES.includes(rawDocketEntry.eventCode);
+      };
```

Comment:
We're adding this compute here, but we noticed that there is similar work going into the `DocketEntry` entity through a PR that is currently awaiting approvals to go to `test`

We would prefer it if we could use the new work going to the `DocketEntry` entity but we can't realistically get that work until it's merged into staging.

Open to thoughts or suggestions from reviewers on how to handle this situation.